common/components/WalletDecrypt/components/DeterministicWalletsModal.tsx

```diff
@@ -8,14 +8,16 @@ import {
 } from 'actions/deterministicWallets';
 import Modal, { IButton } from 'components/ui/Modal';
 import { AppState } from 'reducers';
-import { NetworkConfig } from 'config/data';
+import { NetworkConfig } from 'config';
 import { isValidPath } from 'libs/validators';
 import React from 'react';
 import { connect } from 'react-redux';
 import { getNetworkConfig } from 'selectors/config';
 import { getTokens, MergedToken } from 'selectors/wallet';
 import { UnitDisplay } from 'components/ui';
 import './DeterministicWalletsModal.scss';
+import { DPath } from 'config/dpaths';
+import Select from 'react-select';
```

Comment:
I'm very down with using a select component. We should probably replace most if not all selects with this.